index.bs

```diff
@@ -1,23 +1,22 @@
 <pre class="metadata">
 Shortname: webxr
-Title: WebXR Device API
+Title: WebXR Gamepads Level 1
```

Comment:
Title nit again: Do we want to put the term "Module" in here explicitly? That's what the CSS modules do, and it seems useful to indicate that this is not a standalone spec.